packages/core/src/messagingProtocols/proteus/ProteusService/ProteusService.test.ts

```diff
@@ -124,35 +123,41 @@ const prepareDataForEncryption = async (useQualifiedIds: boolean = true) => {
 describe('ProteusService', () => {
   describe('getRemoteFingerprint', () => {
     it('create a session if session does not exists', async () => {
-      const [proteusService, {apiClient, coreCrypto}] = await buildProteusService();
+      const [proteusService, {apiClient, cryptoClient}] = await buildProteusService();
       const expectedFingerprint = 'fingerprint-client1';
 
-      const getPrekeyMock = jest.spyOn(apiClient.api.user, 'getClientPreKey').mockResolvedValue({
-        client: 'client1',
-        prekey: {
-          id: 123,
-          key: 'pQABARhIAqEAWCCaJpFa9c626ORmjj1aV6OnOYgmTjfoiE3ynOfNfGAOmgOhAKEAWCD60VMzRrLfO+1GSjgyhnVp2N7L58DM+eeJhZJi1tBLfQT2',
-        },
-      });
-      jest.spyOn(coreCrypto, 'proteusFingerprintRemote').mockResolvedValue(expectedFingerprint);
-      jest.spyOn(coreCrypto as any, 'proteusSessionExists').mockResolvedValue(false);
-
```

Comment:
Those scenarios (creating the new session and getting prekeys) are already covered by the SessionHandler tests